.github/ISSUE_TEMPLATE/MaRDI_standard_issue.md

```diff
@@ -15,14 +15,15 @@ Example: "On the website, our Button for accessing the 'about' page is not visib
  
  
 **TODOS**: 
+(In technical terms suggestions what has to be done to fix the issue.)
```

Comment:
This must be inside the HTML comment. Otherwise there is too much text in the tickets, thus one does not see quickly what the issue is about.

Reply:
It is not in comment by intention, TODOS and Acceptance Criteria are easily confused by someone not familiar with the terms.

Reply:
Ok but this is the exact opposite of the intent of this pull request. The PR aims to remove verbose repetitive text to reduce distraction. If someone is not familiar with the term, s/he will learn about it while writing.

Reply:
Some stuff like the examples have been removed already, but this description is good, especially at the start.

Reply:
I disagree. Please and evidence to support your hypothesis.

Reply:
Ok. But the text is still there when you write the issue. The question is, does it help while reading. Maybe one compromise. We can add a link to a project-wide file. Let's name it contributing.md or something similar that explains more details what for example TODOS means. There one can even elaborate that a bit more than just writing
> In technical terms suggestions what has to be done to fix the issue.

This "sentence" alone does not seem helpful to me. I see the following issues:

1. Is it a sentence? What is the verb? Or is it a question?
2. What is the significance of "technical terms"? 
3. Do you mean "on what" instead of "what"
4. Do you need a comma after suggestions?
5.  I think "fix" might not be the best pair with "issue". If it was a "bug" I would understand "fix" however, for issues I would go with resolve. Another alternative is to close and issue. I think this is the label on the button in the GitHub UI.

Overall, I don't see how this sentence contributes to the mission of improving the situation for mathematical research data. I understand that well-written explanations facilitate the creation of high-quality issues. This, in turn, increases our productivity and reduces miscommunication and frustration. However, I am not (yet) convinced that this information needs to be presented to those reading the issues (mainly the MaRDI team), not that the description is good enough to be classified as "well-written".

Reply:
> The question is, does it help while reading

Yes, it does, in my opinion. See explanation above. 

Seriously, if this is about grammatical accuracy, the explanation can be improved in no time. 
How would you phrase the explanation, so that it is short and goes along with your standards ?

Reply:
Something like: 
    In technical terms, suggestions of the tasks to be done to resolve the issue.

Reply:
Explanations file would be a good idea, but the hyperlink will be visible,- at least while editing.

Reply:
To simplify this and end the discussion, I just removed the explanations fields for now. We can re-evaluate this if there is uncertainty about the terms in the template.